src/demo_fake_batteries.cpp

```diff
@@ -0,0 +1,44 @@
+#include "utils/concurrent/thread.hpp"
+#include "sensors/fake_batteries.hpp"
+#include "utils/logger.hpp"
+#include "utils/timer.hpp"
+#include "utils/system.hpp"
+#include "data/data.hpp"
+
+#define FAIL 1
```

Comment:
This is a constant so fake_batteries will always fail? Would a system variable not be better which is set in the command line.